docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -116,7 +116,24 @@ Include it in your module file like this:
 .. code-block:: python
 
     #!/usr/bin/python
-    # Copyright header....
+    # Copyright © 2017 [REPLACE THIS]
+    # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+.. note::
+
+   - Every file with a copyright header must include the original author.
+     Major additions to the module are allowed to add an additional copyright
+     line. This is especially true of rewrites, but original authorship
+     copyright messages should be preserved. If applicable, contributors may
+     have their rights assigned to their employer for example: "Copyright ©
+     2017 Red Hat, Inc."
+
+   - The copyright/license declaration should be ONLY two lines, not the full
+     GPL prefix. If you notice a module with the full prefix, feel free to
+     switch it to the one-line declaration instead.
```

Comment:
Break this into two pieces.  Copyright and license should be separate entries.  Rephrase the second paragraph so it only refers to the license declaration.  Can add a note as to why (not needed + modules are transferred over the wire so extra comments are inefficient.)